modules/solid_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -175,12 +175,12 @@ DomainIntegralAction::~DomainIntegralAction()
 void
 DomainIntegralAction::act()
 {
-  const std::string uo_name("crackFrontDefinition");
-  const std::string ak_base_name("q");
-  const std::string av_base_name("q");
+  const std::string uo_name = "crackFrontDefinition";
```

Comment:
I don't think we mind having constructor syntax for non-integral types. The old way is fine here.